### ISSUE TITLE
Add Gradle example

### ISSUE THREAD

**Issue by scottfrederick** [scottfrederick/springdoclet#3]
Add an example of using SpringDoclet with Gradle.

**Comment by parmindersk** [comment]
``` gradle
configurations {
    springDoclet
}

dependencies {
    springDoclet files("libs/springdoclet-0.0.1-jar-with-dependencies.jar")
}

task generateRestApiDocs(type: Javadoc) {
    source = sourceSets.main.allJava
    destinationDir = reporting.file("rest-api-docs")
    options.docletpath = configurations.springDoclet.files.asType(List)
    options.doclet = "org.springdoclet.SpringDoclet"
}
```

The above configuration is what I used. When I run "gradle generateRestApiDocs" I get this error: javadoc: error - invalid flag: -doctitle

**Comment by thilko** [comment]
What is the state of this issue? Would like to use spring doclet but facing the same issue. Could http://stackoverflow.com/questions/11034326/javadoc-does-not-recognize-doctitle-option-flag/11048833#11048833 be a related issue?

**Comment by sudhcha** [comment]
I've submitted a pull request for this issue - https://github.com/scottfrederick/springdoclet/pull/7. As part of the fix, I changed SpringDoclet to extend from Standard instead of Doclet to get rid of the invalid flag issue mentioned in the issue comments.

Ran tests with a SpringMVC app running on gradle and it worked fine.

**Comment by scottfrederick** [comment]
Fixed in #7.
